src/main/java/org/example/Main.java

```diff
@@ -3,5 +3,6 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        System.out.println("Hello world! again");
```

Comment:
why do u need two line printing the same thing ? 
why do u use print ? why cant  u use logger?

Reply:
i fixed the code please aprpove